hmrc-mongo-play-26/src/main/scala/uk/gov/hmrc/mongo/lock/model.scala

```diff
@@ -18,25 +18,20 @@ package uk.gov.hmrc.mongo.lock
 
 import java.time.LocalDateTime
 
-import play.api.libs.json.{Format, __}
 import play.api.libs.functional.syntax._
-import uk.gov.hmrc.mongo.play.json.formats.{MongoFormats, MongoJavatimeFormats}
+import play.api.libs.json.{Format, __}
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
 
-case class Lock(
-  id         : String,
-  owner      : String,
-  timeCreated: LocalDateTime,
-  expiryTime : LocalDateTime)
+case class Lock(id: String, owner: String, timeCreated: LocalDateTime, expiryTime: LocalDateTime)
 
 object Lock {
 
   implicit val format: Format[Lock] = {
```

Comment:
Similarly for Cache formats, this doesn't need to be implicit and can be moved into the repository.